src/agents/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.13-slim AS builder
```

Comment:
You should add a `base` layer `FROM python:3.13-slim as BASE` and the use `BASE` to not have to repeat `python:3.13-slim`